### DISCUSSION TITLE
Homebrew tap & GitHub releases

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#5120] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

When I create a personal Homebrew tap it creates a new GitHub release for every new formulae/bump up. How Homebrew core tap avoid creating [GitHub releases](https://github.com/Homebrew/homebrew-core/releases)?

**Comment by SMillerDev** [comment, marked as answer]
It uses GitHub Packages. That could work for taps too, but they live on the organisation level so it'll become very messy if you have multiple repos that use them.
